spanner/spanner_snippets/snippet.go

```diff
@@ -80,9 +81,11 @@ func createDatabase(ctx context.Context, w io.Writer, adminClient *database.Data
 	}
 	fmt.Fprintf(w, "Created database [%s]\n", db)
 	return nil
+	// [END spanner_create_database]
```

Comment:
Move the end tags above the final return statement.

Reply:
Done